project/plugins.sbt

```diff
@@ -18,13 +18,13 @@ resolvers += "Typesafe Releases" at "https://repo.typesafe.com/typesafe/releases
 resolvers += "HMRC-open-artefacts-maven2" at "https://open.artefacts.tax.service.gov.uk/maven2"
 resolvers += Resolver.url("HMRC-open-artefacts-ivy2", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
 
-addSbtPlugin("uk.gov.hmrc" %% "sbt-auto-build"     % "3.22.0")
-addSbtPlugin("uk.gov.hmrc"  % "sbt-distributables" % "2.5.0")
+addSbtPlugin("uk.gov.hmrc" %% "sbt-auto-build"     % "3.24.0")
+addSbtPlugin("uk.gov.hmrc"  % "sbt-distributables" % "2.6.0")
 
-addSbtPlugin("org.playframework" % "sbt-plugin" % "3.0.0")
+addSbtPlugin("org.playframework" % "sbt-plugin" % "3.0.3")
 
 addSbtPlugin("org.scalastyle"   % "scalastyle-sbt-plugin" % "1.0.0" exclude ("org.scala-lang.modules", "scala-xml_2.12"))
-addSbtPlugin("org.scoverage"    % "sbt-scoverage"         % "2.0.9")
+addSbtPlugin("org.scoverage"    % "sbt-scoverage"         % "2.3.1")
```

Comment:
This version of scoverage isn't available with Scala 2.13.16, make sure to run with with `coverage test it:test coverageReport` to also check coverage remains consistent and that it compiles